js/carlot.js

```diff
@@ -0,0 +1,64 @@
+var carLot = (function (originalCarLot) {
```

Comment:
Since this IIFE is not an augmentor, and instead an initial IIFE that returns a new object, you don't need to have an argument...